machine/mtrap.c

```diff
@@ -124,6 +124,83 @@ static void send_ipi_many(uintptr_t* pmask, int event)
   }
 }
 
+
```

Comment:
Please move this function to sm/trap.c as well if possible.

Reply:
Just to confirm, but the function send_ipi_many?